src/styles/styles.css

```diff
@@ -0,0 +1,34 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  h1 {
+    @apply text-4xl;
+  }
+  h2 {
+    @apply text-3xl;
+  }
+  h3 {
+    @apply text-2xl;
+  }
+  h4 {
+    @apply text-xl;
+  }
+
+  .default-text-color {
+    @apply text-gray-600;
+  }
+
+  .defatul-bg-color {
+    @apply bg-white;
+  }
+
+  .navbar-text-color {
```

Comment:
instead of adding `navbar` colors here, you can also add this css module inside the `navbar` directory and include it there directly.  (details here: https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css)
Just uses css-modules, so you don't have to add them globally here.

Reply:
Done